tests/__init__.py

```diff
@@ -4,3 +4,9 @@
 ROOT_PATH = Path(__file__).parent.parent
 TEST_CONTENT = "test content"
 ZIP_FILE = "foo.zip"
+ZIP_FILE_ASCII_CONTENT = (
```

Comment:
Is this base64 content? If yes, thinking maybe better to name it `ZIP_FILE_BASE64_CONTENT`?

Reply:
Yeah, I've just copied what our code outputs.